tests/models/whisper/test_modeling_whisper.py

```diff
@@ -235,7 +234,7 @@ def __init__(
         self.num_mel_bins = num_mel_bins
         self.max_position_embeddings = max_position_embeddings
         self.max_source_positions = max_source_positions
-        self.max_target_positions = max_target_positions
+        self.max_target_positions = 448
```

Comment:
Let's revert to a smaller `max_target_positions` and use `config.max_target_positions` in the tests!

Reply:
it's the config that you're using [in the code commented here](https://github.com/huggingface/transformers/pull/33196#discussion_r1744092462)

Reply:
I understood  :)